features/commands.js

```diff
@@ -306,6 +306,20 @@ Here's an article explaining the difference between the two: https://goshakkk.na
 
         fetchMsg.delete();
       }
+    },
+    {
+      words: [`!crosspost`, `!crossposting`],
```

Comment:
maybe we could add a shorter alias? `!cross` perhaps?

Reply:
or
```suggestion
      words: [`!crosspost`, `!xpost`],
```